README.md

```diff
@@ -66,6 +66,8 @@ jobs can be set in one file.
    S3 secret key.
 #### filesystem : str
    ZFS filesystem.
+#### s3_prefix : str, optional
```

Comment:
Could you change the variable name to `prefix` so that it's inline with the other S3 parameters?